src/BigBlueButton.php

```diff
@@ -443,14 +489,59 @@ public function setJSessionId($jSessionId)
         $this->jSessionId = $jSessionId;
     }
 
+    private function isJson($string)
+    {
+        return is_string($string) && is_array(json_decode($string, true)) && (json_last_error() == JSON_ERROR_NONE) ? true : false;
+    }
+
     /* ____________________ INTERNAL CLASS METHODS ___________________ */
```

Comment:
Guess we can remove this line, or the `isJson` should be below this line.

Reply:
ya i will remove that isJson function